### ISSUE TITLE
Unable to Install

### ISSUE THREAD

**Issue by jongravois** [Power-Components/livewire-powergrid#22]
I ran this
`composer require power-components/livewire-powergrid`

and got this
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires power-components/livewire-powergrid ^1.1 -> satisfiable by power-components/livewire-powergrid[v1.1.0].
    - power-components/livewire-powergrid v1.1.0 requires box/spout ^3 -> found box/spout[dev-master, v3.0.0, ..., v3.3.0] but the package is fixed to v2.7.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

How should I proceed???

**Comment by luanfreitasdev** [comment]
Hello @jongravois how are you? 

This problem occurred because you probably have installed a version earlier than the required spout. **but the package is fixed to v2.7.3 (lock file version)** 

I advise updating the spout and you may need to remove the composer.lock file

Please take a test and give me feedback, thanks

**Comment by dansysanalyst** [comment]
Hi @jongravois,

You probably have spot required at 2.7.3 in your `composer.json` file. It should look similar to this:

```
"box/spout": "^2.7.3"`
```

Modify it to:

```
"box/spout": "^3.0",
```

Save it and, once again, run `composer update`. 

This will update your spout, and you should see an output similar to this:

```
  - Downloading box/spout (v3.3.0)
  - Upgrading box/spout (v2.7.3 => v3.3.0): Extracting archive
```
and then run `composer require power-components/livewire-powergrid` and all should be fine  😉

Please let us know if it worked! 

Thanks for downloading!
